x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-studio.adoc

```diff
@@ -0,0 +1,293 @@
+installation directory and change the two lines starting with `-Xms` and `-Xmx` to:
+
+[source,text,linenums]
+----
+-Xms4096m
+-Xmx4096m
+----
+
+Setting the Anypoint Studio memory to 4 GB, which is sufficient
+for the most common uses of this connector.
+////
+// If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:x12-edi-connector-reference.adoc[X12 EDI Reference].
+////
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add X12 EDI (X12 EDI Connector) to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `X12 EDI:Configuration` in the search field.
+. Click X12 EDI:Configuration in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with X12 EDI Connector:
+
+* HTTP Listener +
+Initiates a flow each time it receives a request on the configured host and port.
+
+For example, to configure an HTTP Listener source, follow these steps:
+
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP Listener source in the app.
+. On the *General* tab, specify connection information for the connector.
+. On the *TLS* tab, optionally specify the TLS information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you are specifying an action for that connector to perform.
+
+To add an operation for X12 EDI Connector, follow these steps:
+
+. In the *Mule Palette* view, select *X12 EDI:Configuration* and then select the desired operation.
```

Comment:
```suggestion
. In the *Mule Palette* view, select *X12 EDI* and then select the desired operation.
```

Reply:
These changes are to reflect what is actually in the UI when you go through these steps. I installed X12 Connector and am verifying the UI.